.github/workflows/ci.yml

```
@@ -35,6 +35,59 @@
+    name: Build and Tag Docker Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@2b51285047da1547ffb1b2203d8be4c0af6b1f20
+
+      - name: Get version from package.json
+        id: package-version
+        run: echo "VERSION=$(node -p "require('./packages/api-reference/docker/package.json').version")" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:96: Double quote to prevent globbing and word splitting

(shellcheck)